Doc/library/calendar.rst

```diff
@@ -170,6 +170,67 @@ it's the base calendar for all computations.
       sheet should be used. *encoding* specifies the encoding to be used for the
       output (defaulting to the system default encoding).
 
+   :class:`HTMLCalendar` has the following attribute you can override to customize the style of your calender:
+
+  .. attribute:: cssclasses
+
+    Is a list of CSS styles used for each weekday. The default style list is::
```

Comment:
Not sure this should start with 'is'.  The calendar attributes on this page all start with a subject, such as 'An array'.